05/app.js

```diff
@@ -11,15 +11,12 @@ const stats = {
 const paragraphList = document.querySelectorAll('p');
 
 const countClicks = function (element, obj) {
-    let counter = 0;
-
     if (!isNaN(obj[element])) {
-        obj[element] = ++obj[element];
+        obj[element] += 1;
     } else {
-        obj[element] = ++counter;
+        obj[element] = 1;
     }
 }
```

Comment:
Teraz można to uprosić - raczej ja bym tak zrobił:
```
if (isNaN(obj[element])) {
    obj[element] = 0;
}

obj[element]++
```
PS. Zwróć uwagę, że nie pisałem tego wcześniej bo nie było tego widać. Zmiany robimy krok po kroku. Wtedy łatwiej jest je wychwycić i mniej można zepsuć ;)

Reply:
Faktycznie, wygląda to lepiej!
Poprawione – dziękuję ;).